src/components/amount/amount.ts

```diff
@@ -50,6 +50,7 @@ export class AmountComponent implements OnInit {
 
   public onInputToken() {
     const precision = this.marketCurrency.code === 'btc' ? 8 : 2;
+    this.amount = +(1.0 * this.amount).toFixed(8);
```

Comment:
We should use the `BigNumber` package and the `ARKTOSHI_DP` constant. Take a look: https://github.com/ArkEcosystem/mobile-wallet/blob/62163ecd1e585eb7689ce7f34667cd2bd44a02d9/src/utils/ark-utility.ts#L13